source/_default_sidebar.md.erb

```diff
@@ -14,15 +14,14 @@ Older [meetings](/meetings/) \| [nights](/nights/) \| [podcasts](/podcasts/)
 
 <%= meeting_calendar_link %>
 
-### Get In Touch
+### Community
 
 * [Mailing List](http://lrug.org/mailing-list)
 * [IRC](/irc/) (#lrug on freenode)
 * [El Rug (Twitter)](http://twitter.com/lrug)
 * Photos: [group (preferred)](http://flickr.com/groups/680991@N25/) or [tag](http://flickr.com/photos/tags/lrug/)
```

Comment:
I kept Flickr as the links still work, but they haven't seen any updates for since around 2011.